_apps/rtd-bot.md

```diff
@@ -0,0 +1,20 @@
+---
+# A human-friendly name of your listing
+title: Read The Docs
+# A short description of what your app does
+description: An unofficial GitHub Probot that helps reveiwing on Read The Docs
```

Comment:
We prefer to drop Probot/GitHub form the description since all the apps on the website are such. What about:

> Assists reviewing using Read The Docs